pkg/db/sqlc/queries/jobs.sql

```diff
@@ -11,9 +11,10 @@ select
     ,(select count(*) from applications a where a.job_id = j.id) as application_count
     ,(CASE WHEN p.display_name = '' THEN p.login ELSE p.display_name END)::varchar AS customer_display_name
     ,p.ethereum_address AS customer_ethereum_address
+    ,j.visibility
```

Comment:
I checked it, maybe it would be good to remove this column from this query, because we will not use it on frontend.

Reply:
And after this please execute `make sqlc` in the project root folder to update sqlc models 🙏

Reply:
ok will do the change

Reply:
updated

https://github.com/optriment/optrispace-backend/pull/143/commits/fc8914b164d646e258872cddcb6b0e736fbcf90b